modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -11,7 +11,80 @@ Before you install an {product-title} cluster, review the following Nutanix AOS
 [id="installation-nutanix-installer-infra-reqs-account_{context}"]
 == Required account privileges
 
-Installing a cluster to Nutanix requires an account with administrative privileges to read and create the required resources.
+The installation program requires access to a Nutanix account with the necessary permissions to deploy the cluster and to maintain the daily operation of it. The following options are available to you:
+
+* You can use a local Prism Central user account with administrative privileges. Using a local account is the quickest way to grant access to an account with the required permissions.
+* If your organization’s security policies require that you use a more restrictive set of permissions, use the permissions in the following table to create a custom Cloud Native role in Prism Central. You can then assign the role to a user account that is a member of a Prism Central authentication directory.
```

Comment:
Super nit 
>use the permissions in the following table

This _almost_ sounds like the table is where you use the permissions. Or am I being too picky? 

>use the can opener in kitchen 

Is that where to use the can opener or where the can opener is? 

> use the permissions listed/defined in the table